### ISSUE TITLE
is it save to remove LDAP for part Running/configuration-with-ocis in in documentation?

### ISSUE THREAD

**Issue by MonksterFX** [owncloud/ocis-hello#93]
So I followed the [guide](https://github.com/owncloud/ocis-hello/blob/master/docs/configuration-with-ocis.md) along and I experienced that it is a little bit hard to follow. So I want to help to improve it a little bit from a beginners view. 

I recognized that LDAP is not necessary at all, because **Kopano** was added now in the main ocis repro. 

This would bring down the complexity of this part of the guide a lot. I wanted to ask before I submit changes.

**Comment by kulmann** [comment]
Hey, thank you for taking the time! Good spot, LDAP is not a requirement anymore, because we use our accounts service as user backend. It is still possible to use LDAP, but for the `hello` docs it is out of scope.

For easier reading, you can find a rendered version of the documentation here: https://owncloud.github.io/extensions/ocis_hello/

**Comment by wkloucek** [comment]
LDAP dependencies in the `go.sum` file are only transitive dependencies (REVA / oCIS) and cannot be removed, but there is also no need to do so from my viewpoint since they are only transitive. `package.json` explicitly has `ldap` and `ldapjs` as an devDependency which could just originate from the ownCloud Web repo (copy-paste)!?

**Comment by pascalwengerter** [comment]
> LDAP dependencies in the `go.sum` file are only transitive dependencies (REVA / oCIS) and cannot be removed, but there is also no need to do so from my viewpoint since they are only transitive. `package.json` explicitly has `ldap` and `ldapjs` as an devDependency which could just originate from the ownCloud Web repo (copy-paste)!?

Just checked `web` and we're only using `ldapjs` as devDependency/for acceptance tests. Let me take a look later by linking stuff locally and see if it still works ™️  so we can eventually remove them from ocis-hello!
